### PR TITLE
update messages, links and constants

### PR DESCRIPTION
**Pull request by giteden**
## Proposed Changes

- update links to new docs (bit.dev/docs) whenever possible
- replace BASE_WEB_DOMAIN with BASE_CLOUD_DOMAIN and BASE_COMMUNITY_DOMAIN
- update registry entry to node.bit.cloud

### CODE REVIEW

File: scopes/component/component-issues/multiple-envs.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { ComponentIssue, ISSUE_FORMAT_SPACE } from './component-issue';
 export class MultipleEnvs extends ComponentIssue {
   description = 'multiple envs';
   solution =
-    'remove the old envs by setting them with "-" sign in the variants. see https://harmony-docs.bit.dev/aspects/variants#removing-aspects ';
+    'remove the old envs by setting them with "-" sign in the variants. see https://bit.dev/docs/envs/using-envs#component-must-have-a-single-env';
```
Comment (GiladShoham) [inline scopes/component/component-issues/multiple-envs.ts:6]: Use base docs domain constant

**Review by GiladShoham — CHANGES_REQUESTED** [review]

File: scopes/component/component-issues/multiple-envs.ts (hunk 1 above)
Comment (giteden) [reply scopes/component/component-issues/multiple-envs.ts]: done (replaced all instances of hardcoded docs domain with the docs domain constant)

File: scopes/dependencies/dependency-resolver/dependency-resolver.main.runtime.ts [hunk 2]
```diff
@@ -77,7 +77,11 @@ import { DependencyDetector } from './dependency-detector';
 import { DependenciesService } from './dependencies.service';
 import { EnvPolicy, EnvPolicyFactory } from './policy/env-policy';
 
-export const BIT_DEV_REGISTRY = 'https://node.bit.dev/';
+/**
+ * @deprecated use BIT_CLOUD_REGISTRY instead
+ */
+export const BIT_DEV_REGISTRY = 'https://node.bit.cloud/';
```
Comment (GiladShoham) [inline scopes/dependencies/dependency-resolver/dependency-resolver.main.runtime.ts:83]: this should point to the old registry

**Review by GiladShoham — CHANGES_REQUESTED** [review]

File: scopes/harmony/cli/help.ts [hunk 3]
```diff
@@ -72,7 +72,7 @@ function commandTemplate(name: string, description: string): string {
 function getHeader(): string {
   return `${chalk.bold('usage: bit [--version] [--help] <command> [<args>]')}
 
-${chalk.yellow('bit documentation: https://harmony-docs.bit.dev')}`;
+${chalk.yellow('bit documentation: https://bit.dev/docs')}`;
```
Comment (GiladShoham) [inline scopes/harmony/cli/help.ts:75]: use constant

File: src/cli/commands/public-cmds/init-cmd.ts [hunk 4]
```diff
@@ -16,9 +16,9 @@ import { CommandOptions, LegacyCommand } from '../../legacy-command';
 export default class Init implements LegacyCommand {
   name = 'init [path]';
   skipWorkspace = true;
-  shortDescription = 'create or reinitialize an empty Bit scope or reinitialize an existing one';
+  shortDescription = 'create or reinitialize an empty workspace';
```
Comment (GiladShoham) [inline src/cli/commands/public-cmds/init-cmd.ts:19]: this used also to init a scope (with the bare flag)

**Review by GiladShoham — CHANGES_REQUESTED** [review]

File: scopes/dependencies/dependency-resolver/dependency-resolver.main.runtime.ts (hunk 2 above)
Comment (giteden) [reply scopes/dependencies/dependency-resolver/dependency-resolver.main.runtime.ts:83]: done

File: scopes/harmony/cli/help.ts (hunk 3 above)
Comment (giteden) [reply scopes/harmony/cli/help.ts:75]: done

File: src/cli/commands/public-cmds/init-cmd.ts (hunk 4 above)
Comment (giteden) [reply src/cli/commands/public-cmds/init-cmd.ts:19]: the description should describe the main functionality and should not include its various options. 
'bit init --bare' is just one way of initializing a workspace (bare of the working directory)

**Comment by GiladShoham** [comment]
replaced by #5544

**Pull request closed without merging**